### DISCUSSION TITLE
Taking somebodys username

### DISCUSSION THREAD

**Discussion by Blaumaus** [community/community#21695] [New to GitHub]
<p>Following <a href="https://help.github.com/en/articles/name-squatting-policy" rel="nofollow noopener">Name Squatting Policy</a> how much time does account need to be inactive to release it’s username?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/ookldev">@ookldev</a>.,</p>
<p>Thanks for being here! I’m afraid we don’t share specific details regarding our thresholds for account dormancy. However it is worth keeping in mind that just because an account doesn’t have any publicly visible activity, this doesn’t necessarily mean that they don’t have any activity at all.</p>
<p>If you haven’t already, please contact us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> with the info so we can look into your specific case further.</p>
